src/main/scala/com/fortysevendeg/android/sqlite/metadata/SQLDroidDatabaseMetaData.scala

```diff
@@ -0,0 +1,933 @@
+package com.fortysevendeg.android.sqlite.metadata
+
+import java.sql._
+
+import android.database.{MatrixCursor, MergeCursor}
+import com.fortysevendeg.android.sqlite.metadata.SQLDroidDatabaseMetaData._
+import com.fortysevendeg.android.sqlite.resultset.SQLDroidResultSet
+import com.fortysevendeg.android.sqlite.util.CursorUtils._
+import com.fortysevendeg.android.sqlite.{SQLDroidDriver, WrapperNotSupported, _}
+import org.sqldroid.SQLDroidConnection
+
+import scala.util.{Failure, Success, Try}
+
+class SQLDroidDatabaseMetaData(connection: Connection)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/784801318/issues/source?bid=2442814&fileBranchId=2493741#l14)